admin_manual/configuration/server/config_sample_php_parameters.rst

```diff
@@ -819,7 +819,7 @@ seen in the first-run wizard and on Personal pages.
 Apps
 ----
 
-Options for the Apps folder, Apps store, and App code checker.
+Options for the Apps store, Apps folder, and App code checker.
```

Comment:
Why is this change needed, just curiously?

Reply:
The new sort order is intentionally because it´s occurence follows text to description:

Apps store -> 'appstoreurl'
Apps folder -> 'apps_paths'
App code -> this one is missing and another story...

Reply:
Thanks for the clarification.